core/frontend/src/render/webgl/SceneCompositor.ts

```diff
@@ -2012,31 +2029,37 @@ class MRTCompositor extends Compositor {
       this.drawPass(commands, RenderPass.OpaqueLinear);
       this.drawPass(commands, RenderPass.OpaquePlanar, true);
       if (needAO || renderForReadPixels) {
+        if (needAO)
```

Comment:
Image Tests shows numerous AO differences.  I removed the shader change, but still many (different) differences.  Will need to investigate further.

Reply:
Cause: some non-pickable things draw and still output kRenderOrder in the pick buffer, which AO shaders use to cull out certain things like lines, etc. from the algorithm.  Skipping drawing these means they get skipped when applying AO because their kRenderOrder is not set.

Reply:
@MarcNeely and I pushed a possible fix for this.  Will need to run Image Tests again.